BCStudentSoftwareDevTeam/celts
# Volunteer duplicates

Fixes #735 

Removed the duplicate rows of a volunteer that occurs each time the event is in the past. Instead of having multiple duplicated rows for a single volunteer, we now have a single row.

To test:
Navigate to the Event List: IP_Address/eventsList/1/track_volunteers.  Click on edit event to confirm if event has already occurred. If yes, click on Track Volunteers and for the same volunteer that is on view, it will show only one row per volunteer. Also, you can add a new volunteer for a past event and only one row would be created per volunteer.
